README.md

```diff
@@ -25,6 +25,144 @@ Follow the instructions in [demo/README](demo/README.md) to set up an example Dj
 
 ---
 
+# Install and use it
+
+[Start a new Django project](https://docs.djangoproject.com/en/2.2/intro/tutorial01/), if you don't have one already:
+```
+$ django-admin startproject myincidentresponse
+```
+
+Install response:
+```
+$ pip install django-incident-response
+```
+
+In `settings.py`, add these lines to `INSTALLED_APPS`:
+```
+INSTALLED_APPS = [
+    ...
+    "response.ui.apps.UiConfig",
```

Comment:
Is the order important here?  Asking as it'd _look_ nicer if ther response ones were together 🙂

Reply:
I'm not actually sure - I've just copied the existing ordering :) if you're testing these instructions, perhaps try them the other way round?